jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletContextHandler.java

```diff
@@ -1196,7 +1196,14 @@ protected boolean handleByContextHandler(String pathInContext, ContextRequest re
         boolean initialDispatch = request instanceof ServletContextRequest;
         if (!initialDispatch)
             return false;
-        return super.handleByContextHandler(pathInContext, request, response, callback);
+
+        if (isProtectedTarget(pathInContext))
+        {
+            // Do nothing here other than set the error status so that the ServletHandler will handle as if a sendError
+            request.setAttribute(org.eclipse.jetty.server.handler.ErrorHandler.ERROR_STATUS, 404);
+            response.setStatus(HttpStatus.NOT_FOUND_404);
+        }
+        return false;
```

Comment:
This is now different to `ee11` - is that on purpose or an omission?

Reply:
The changes in ee11 required a change to the ErrorPageMapper API and implementation, that I do not want to do in EE10, so just doing the simplest solution here.   In EE11, I have tried to clean up the ErrorPageMapper API somewhat... but I think more could be done (maybe a different PR?)